pkg/apis/pipeline/images.go

```diff
@@ -21,6 +21,13 @@ import (
 	"sort"
 )
 
+// Options holds options passed to the Tekton Pipeline controllers
+// typically via command-line flags.
+type Options struct {
```

Comment:
`pkg/apis/pipeline` is a bit of a "util" / "bag of random stuff" package (storage options, path constants, images struct, gvk definitions, etc). Given that, I suggest putting the `Options` struct into its own `controller_options.go`, or renaming `images.go`. Ideally `images.go` would only contain code related to `Images`. Also suggest bringing the field comments over from the `ControllerConfiguration` type we're replacing for that same reason - additional context in an otherwise context-less package.

Reply:
Good call, done.